src/app/download/download.service.ts

```diff
@@ -9,6 +9,16 @@ export class DownloadService {
     private readonly adAuthoringState: AdAuthoringWorkflowStateContainer
   ) {}
 
+  getDisabledVal() {
```

Comment:
nit: I'd prefer a more clear name, maybe downloadDisabled(). Val is ambiguous here.